src/session/content/item_row.rs

```diff
@@ -31,11 +31,10 @@ mod imp {
                     "item",
                     "Item",
                     "The item represented by this row",
-                    Item::static_type(),
+                    glib::Object::static_type(),
                     glib::ParamFlags::READWRITE,
```

Comment:
I would either add a `glib::ParamFlags::CONSTRUCT_ONLY` here and notify in the `ItemRow::set_item()` function
or make the `ItemRow::set_item()` function private.
Else, this could cause a bug if we in the future, for whatever reason, call `ItemRow::set_item()` directly.

Reply:
Yeah, I forgot about that.